### PR TITLE
Math magicians: full website 👌🏼

### PR DESCRIPTION
**Pull request by TadesseDev**
# Changes introduced 😎
---
- Create a website consisting of 3 pages: Home, Calculator, and Quote. ✔
- Add styles to improve the look and feel of the website. ✔
- Create a route for each page. ✔

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export default function Navbar() {
+  // [{ name: Home, link: '/' }];
```
Comment (Deepakdanger) [inline src/components/Navbar.js]: - [ ] [Optional] Kindly remove the commented code.

**Review by Deepakdanger — APPROVED** [review]
Hi @Tadesse-Alemayehu, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] _Check the comment below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.js (hunk 1 above)
Comment (TadesseDev) [reply src/components/Navbar.js]: ### Thank you @Deepakdanger I'll make sure to correct the optional suggestion before maraging the pull request.

**Comment by TadesseDev** [comment]
### Dear @Deepakdanger Changes are applied in https://github.com/Tadesse-Alemayehu/Math-Magicians/pull/5/commits/3b4953488b4b5d6f62ae2754bb48533b5370ecd2 commit hash. Feel free if there are suggestions still to be improved.

**Pull request merged**